.clang-tidy

```diff
@@ -0,0 +1,12 @@
+Checks:
+  *,
+  -altera*,
```

Comment:
I am assuming this enables all the tests except for those listed with `-*`.

But to disable the Altera tests, isn't it `-altera-*` according to [1]? Maybe the `*` makes up for the `-` too.

### References
[1] https://clang.llvm.org/extra/clang-tidy/

Reply:
Yes on the first point.
On the second point, in practice, the * works anywhere in the check name.